stories/utils/features.js

```diff
@@ -33,7 +29,7 @@ function setFeatureActive( feature, active ) {
  * @param {string} feature Feature to enable.
  */
 export function enableFeature( feature ) {
-	setFeatureActive( feature, true );
+	set( global, `_googlesitekitBaseData.enabledFeatures.${ feature }`, true );
```

Comment:
This whole module should be removed per the ACs since feature activation would be entirely handled by the new provider for stories.